composer.json

```diff
@@ -32,5 +23,9 @@
     "psr-0": {
       "Curl": "src/"
     }
+  },
+  "support": {
+    "source": "https://github.com/php-curl-class/php-curl-class",
```

Comment:
This library is a fork of php-curl-class from November 3, 2013. They are no longer linked anymore.

https://github.com/php-mod/curl/commit/747df2b934fc29ab1f5b710cf6c43374bf25f6ac